mantidproject/mantid
# Sequential fit in muon analysis

Browse would in muon analysis' sequential fit dialog would crash mantid
**To test:**
Open Muon analysis
Load HIFI 108372
Go to data analysis and add a fitting function
Do a fit
Then go to fit-> sequential fit
In the Runs box have: 108372-3,108376
Click start (it should work)
Close the dialog
Turn off the data archive and add the path to your muon data in the search directory path
Do another sequential fit but this time select your muon data with the browse button
It should work

Remove the muon path from the search directory path see #22042



Fixes #22020. 

**Release Notes** 

In release notes
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
